### PR TITLE
Print out cron exceptions

### PR DESCRIPTION
**Pull request by convenient**
Magerun pull-request check-list:
- [x] Pull request against develop branch (if not, just close and create a new one against it)
- [x] README.md reflects changes (if any)

I was running a cron via the CLI yesterday and didn't realise it was throwing an exception. This change makes it so that exceptions are output to the command line.

Output Before:

```
Run Namespace_Module_Model_Cron::run
done
```

Output After:

```
Run Namespace_Module_Model_Cron::run
                                                 
  [Mage_Core_Exception]                          
  Exception Message Here
                                                 
sys:cron:run [job]
```

Running the command with `-v` will also print out the stack trace.

### CODE REVIEW

**Comment by tkn98** [comment]
Sounds legit. Can you share the exit status code as well? I think it's useful that it is non-zero.

File: src/N98/Magento/Command/System/Cron/RunCommand.php [hunk 1]
```diff
@@ -94,6 +94,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     ->setFinishedAt(strftime('%Y-%m-%d %H:%M:%S', time()))
                     ->save();
             } catch (Exception $e) {
+                $output->writeln('');
```
Comment (tkn98) [inline src/N98/Magento/Command/System/Cron/RunCommand.php]: There is the \Symfony\Component\Console\Application::renderException method, see src/N98/Magento/Application.php:602 for an example, I think it only needs little adoption for a command.

Comment (convenient) [inline src/N98/Magento/Command/System/Cron/RunCommand.php]: Neato, will give it a gander shortly 👍

**Comment by tkn98** [comment]
@convenient: I don't want you to push you into any direction nor burn your time. Just let me know if you want to/can progress with this issue or not. Would be good to know for release planning.

**Comment by convenient** [comment]
Damn, I completely forgot about this with SUPEE8788. Just force pushed up my new change. 

Super small change which works well with the rest of the application, returns a non-zero exit code thanks to the error handling in `Symfony\Component\Console\Application::run`.

**Comment by tkn98** [comment]
Now that's a commit I can't refuse to immediately take in ;) Thanks a lot.

**Comment by tkn98** [comment]
Just re-classified this as a bug as this should provide the non-zero exit code.

We believe that the bug reported is fixed in the development version. It can be upgraded to it using the `self-update` command with the `--unstable` switch.

**Comment by convenient** [comment]
Thanks dude 👍

**Pull request merged**